auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -329,15 +347,27 @@ public int hashCode() {
 
         @Override
         public String toString() {
+            StringBuilder paramsValueBuilder = new StringBuilder();
+            paramsValueBuilder.append("{");
+            for (String key : mParams.keySet()) {
+                paramsValueBuilder.append(key);
+                paramsValueBuilder.append("=");
+                paramsValueBuilder.append(mParams.get(key).toString());
+                paramsValueBuilder.append(",");
+            }
+            paramsValueBuilder.append("}");
```

Comment:
Doesn't `Bundle` already have a `toString()` with very similar behavior?

Reply:
Yeah, just check the source and `Bundle#toString()` calls `mMap.toString()`. I'm assuming the formatting will be a little bit different, but the output should be effectively the same.

Reply:
Ah, did not realize that. Switched to the Bundle toString.